### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by mjpost**
Here I plan to update the ingestion documentation, switching from a spreadsheet to an ingestion template, which will make things easier for us to track.

I vaguely recall a recent discussion about simplifying the correction templates (metadata vs. PDF), too, but I can't seem to find it. Maybe you remember, @akoehn?

### CODE REVIEW

**Comment by akoehn** [comment]
> I vaguely recall a recent discussion about simplifying the correction templates (metadata vs. PDF), too, but I can't seem to find it. Maybe you remember, @akoehn?

We have #1184 regarding author list changes.

**Comment by mjpost** [comment]
This has been held up way too long. I split off the issue templates into #1826, so that this mostly just contains documentation and other site cleanup (such as removing the request for presentations: #1401).

File: hugo/content/info/ids.md [hunk 1]
```diff
@@ -2,35 +2,39 @@
...
+In addition to paper identifiers, there are a number of other useful concepts.
+
+Volumes themselves also have identifiers.
+A volume identifier is formed by removing the paper number and the delimiter, e.g., `2020.acl-main`.
+Volumes can be accessed via the same scheme for canonical pages used for papers, e.g., <https://aclanthology.org/2020.acl-main>.
+
+We also infer the presence of events.
+Each yearly entry for a venue creates an "events" page.
+This can be accessed using the year and venue code.
+For example, the ACL 2020 event is available at <https://aclanthology.org/eventes/acl-2020>.
```
Comment (mbollmann) [inline hugo/content/info/ids.md:35]: typo in the URL ("eventes")

File: hugo/content/_index.md [hunk 2]
```diff
@@ -6,7 +6,3 @@ css_container: container-fluid
 
 <!-- To disable printing a date for the MOTD, set the "date" field above to some
 value before 2019. -->
-
```
Comment (mbollmann) [inline hugo/content/_index.md:9]: This results in an empty blue box now (can be seen on the preview's front page).

You might want to comment out these lines in the template if we don't want to use this box for anything right now: https://github.com/acl-org/acl-anthology/blob/b7d4380974fceda2deff4af08b0f13eaad93f565/hugo/layouts/index.html#L37-L43

**Comment by mjpost** [comment]
Thanks for catching the typos and making the corrections easy!

**Pull request merged**